### DISCUSSION TITLE
Galaxy buds Pro

### DISCUSSION THREAD

**Discussion by geekmaster21** [d4rken-org/capod#370] [Ideas]
Hey, the app dont detect my galaxy buds pro !

**Comment by d4rken** [comment]
If you tell me how to decode their Bluetooth data, then I'll add support :slightly_smiling_face:

**Comment by amosstevens** [comment]
maybe this can help

https://github.com/timschneeb/GalaxyBudsClient

**Comment by d4rken** [comment]
@amosstevens That would help, thanks! 

Looks like it's very different from AirPods. Dose it make sense to include it in an "Companion for AirPods" app? 🤔 Silent functionality? Rebrand the whole app?

Re-opening for now 👍
